modules/mo_pytorch/test/requirements.txt

```diff
@@ -1,4 +1,4 @@
-torch==1.9.1
+torch==1.13.1
```

Comment:
Shouldn't we enable tests for this module before changes?

Reply:
ok, who can add tests? :)

Reply:
They are disabled in https://github.com/openvinotoolkit/openvino_contrib/pull/384

Reply:
I disabled the mo_pytorch tests because at the time mo_pytorch was not being maintained at that time and the failing tests caused CUDA plugin tests to fail. If mo_pytorch is now maintained again the tests should be enabled again.

Reply:
I don't think that support of that experimental module will be continued at this moment. Especially considering https://github.com/pytorch/ort .

Reply:
should we remove the code and refer to new approach?

Reply:
Final decision is yours but due to limited resources for maintaining and available alternative which probably can produce IRs as well, it make sense to keep just on the releases branch.

Reply:
Sure, I will. Maybe let's change just requirements file for now to make dependencies check happy. With no tests.

Reply:
reverted changes in CI files